save-reporters/src/commonMain/kotlin/org/cqfn/save/reporter/json/JsonReporter.kt

```diff
@@ -73,10 +73,11 @@ class JsonReporter(override val out: BufferedSink) : Reporter {
      *
      * @return `true` if comma has been written, `false` otherwise
```

Comment:
Please also fix this doc: with current implementation, after the first invocation of this method flag should be switched to false, and we re-assign true somewhere else.

Reply:
Thanks, fixed